### PR TITLE
Move systemtags (app) to webpack

### PR DESCRIPTION
**Pull request by rullzer**
* Moves icon to icons.scss
* Bundle js with webpack
* Bundles the scss with webpack

Gets rid on 1 request (css). But the big win is again that it is served statically by the webserver

### CODE REVIEW

**Comment by rullzer** [comment]
Even green CI! Review time!

**Review by ChristophWurst — APPROVED** [review]
Looks good!

File: core/css/icons.scss [hunk 1]
```diff
@@ -479,7 +479,10 @@ img, object, video, button, textarea, input, select, div[contenteditable='true']
 	@include icon-color('search', 'actions', $color-black, 1, true);
 }
 
-
 .icon-talk {
 	@include icon-color('app-dark', 'spreed', $color-black, 1);
 }
+
+.nav-icon-systemtagsfilter {
+	@include icon-color('tag', 'systemtags', $color-black);
```
Comment (juliushaertl) [inline core/css/icons.scss]: We can just use the icon from core here and remove the one from apps/systemtags/img/tag.svg

```suggestion
	@include icon-color('tag', 'actions', $color-black, 1, true);
```

**Review by juliusknorr — CHANGES_REQUESTED** [review]

File: core/css/icons.scss (hunk 1 above)
Comment (juliushaertl) [reply core/css/icons.scss]: Actually the class doesn't seem to be used anywhere, so maybe we can just get rid of it.

Comment (rullzer) [reply core/css/icons.scss]: It is used. Generated dynamically. Inspect the tags icon in the left sidebar

Comment (rullzer) [reply core/css/icons.scss]: But resolved :D

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**